CONTRIBUTING.md

```diff
@@ -72,7 +77,7 @@ $ docker-compose up
 
 #### Aliases
 
-Some aliases are conveniently made available... 
+Some aliases are conveniently made available…
```

Comment:
I think you also used the makefile whereas it wasn't mentioned anywhere.
Could you add a note on it somewhere?

Reply:
I did not use the makefile, since I threw the towel and went for local docker dev.

Reply:
Alright thx for the feedback.